# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: js/xpconnect/tests/mochitest/mochitest.ini
# Commit: 490dbdb4e499
# Full Hash: 490dbdb4e4996b91ab7aa8b4dd70de4029602c73
# Author: yulia <ystartsev@mozilla.com>
# Date: 2021-01-12 04:33:40
# Regressor Bug: 1681664
# File Overlap Count: 2
# Description:
#   Bug 1681664 - Allow Top-level await modules which fail to evaluate due to user action to fail without crashing. r=jonco,mconley
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D99809
# ==============================================================================

diff -r 53327022537a -r 490dbdb4e499 js/xpconnect/tests/mochitest/mochitest.ini
--- a/js/xpconnect/tests/mochitest/mochitest.ini	Mon Jan 11 16:53:32 2021 +0000
+++ b/js/xpconnect/tests/mochitest/mochitest.ini	Mon Jan 11 17:08:59 2021 +0000
@@ -38,9 +38,11 @@
   test2_bug629331.html
   finalizationRegistry_worker.js
   private_field_worker.js
+  bug1681664_helper.js
 prefs =
   javascript.options.weakrefs=true
   javascript.options.experimental.private_fields=true
+  javascript.options.experimental.top_level_await=true
 
 [test_bug384632.html]
 [test_bug390488.html]
@@ -99,6 +101,7 @@
 [test_bug1094930.html]
 [test_bug1158558.html]
 [test_bug1448048.html]
+[test_bug1681664.html]
 [test_crosscompartment_weakmap.html]
 [test_enable_privilege.html]
 [test_frameWrapping.html]
@@ -123,4 +126,4 @@
 [test_weakRefs_cross_compartment.html]
 [test_weakRefs_collected_wrapper.html]
 [test_private_field_dom.html]
-[test_private_field_worker.html]
\ No newline at end of file
+[test_private_field_worker.html]